src/ExerciseOOP/Library.java

```diff
@@ -1,4 +1,77 @@
 package ExerciseOOP;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Library {
+
+    public static void main(String[] args) {
+        List<Book> listBook = new ArrayList<>();
+        Book book1 = new Book("Doanh Nghiep The Ki 21", "Robert", "1/1/2000", "Available");
+        Book book2 = new Book("NDCD", "Bob Burg", "1/1/2000", "Available");
+        Book book3 = new Book("NLDKCD", "Robin Sharma", "1/1/2000", "Available");
+        listBook.add(book1);
+        listBook.add(book2);
+        listBook.add(book3);
+        System.out.println("---------MENU------------");
+        System.out.println("1. Add book");
+        System.out.println("2.Search book");
+        System.out.println("3. Borrow book");
+        System.out.println("4. Return book");
+        System.out.println("5. Display book");
+        System.out.println("6. Exit");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("What do you want?. Please type the number as the menu");
+        int userType = scanner.nextInt();
+        if (userType == 1) {
+            listBook = addBook();
+            System.out.println(listBook.get(0).toString());
```

Comment:
Each user function should be move to a function. For example, these code should be in a handleAddBook